TROUBLESHOOTING.md

```diff
@@ -190,3 +191,23 @@ ENOGIT
 ### Related Issues
 
 * [#11095](https://github.com/npm/npm/issues/11095)
+
+## Cannot Read property
+If your install fails and it says something like `Cannot read property 'path' of null`, then it is most likely a network issue in disguise.
```

Comment:
I'm... not really comfortable putting this in as a general troubleshooting item because that error can be caused by a number of things. (Ultimately it is, of course, an error in npm itself.) I might feel better about it if it were to cite versions where it's been known to happen?